src/elements/toast/Toast.js

```diff
@@ -0,0 +1,56 @@
+import { Component, ElementRef } from 'angular2/core';
+import { CORE_DIRECTIVES } from 'angular2/common';
+
+@Component({
+    selector: 'novo-toast',
+    inputs: ['theme', 'icon', 'title', 'message', 'time', 'position'],
```

Comment:
Should `show` be an input? Or will the user user `*ngIf` to hide/show the toast?